src/main/java/com/hms/flexyosisoftconnector/OSIsoftConnectorMain.java

```diff
@@ -72,6 +72,12 @@ public static void main(String[] args) throws IOException, JSONException {
     // update tag info to use later
     TagInfoManager.refreshTagList();
 
+    if (TagInfoManager.getTagInfoList().size() == 0) {
+      Logger.LOG_SERIOUS("There are 0 tags monitored for telemetry data to OSIsoft.");
+      Logger.LOG_SERIOUS("Add 1 or more tags to telemetry data and restart the connector. Connector application exiting.");
+      System.exit(0);

```

Comment:
This is *ok* for now, but I would much rather see the actual code below wrapped in this if. (maybe check if TagInfoManager.getTagInfoList().size() > 0, and have this logging in the else statement).

Also, System.exit(int) with a value of zero (`0`) implies that the application exited cleanly/successfully. If there is a serious message or issue to convey to the user, exiting with `0` implies the opposite.

Reply:
Changed to exit code 1 instead of 0. It closes the application so there shouldn't be a need to wrap the code below, the application would not be running.